ruth/tools/simulator.py

```diff
@@ -48,11 +48,33 @@ class AlternativesRatio:
 
     def __post_init__(self):
         self.default = 1 - self.dijkstra_fastest - self.dijkstra_shortest - self.plateau_fastest
+        self.default = round(self.default, 2)
         if self.default < 0:
             raise ValueError("Sum of alternatives ratios must be equal to 1.")
 
+    def to_list(self):
+        return [self.default, self.dijkstra_fastest, self.dijkstra_shortest, self.plateau_fastest]
 
-def prepare_simulator(common_args: CommonArgs, vehicles_path, alternatives_ratio: AlternativesRatio)\
+
+@dataclass
+class RouteSelectionRatio:
+    no_alternative: float
```

Comment:
What does `no_alternative` mean?

Reply:
We added a description of this to README and enum definition. Is it sufficient like this?

Reply:
Yeah, thanks :) An assert somewhere that `no_alternative` is indeed used with `default` would be nice, but now I understand what it means.